src/Contrl/Output.f90

```diff
@@ -20,11 +20,12 @@ module Outputclass
       contains
         !> calculate <x^2>,<xp>,<px^2>,x emittance, <y^2>,<ypy>,
         !> <py^2> and y emittance, <z^2>,<zp>,<pz^2>,z emittance.
-        subroutine diagnostic1_Output(z,this)
+        subroutine diagnostic1_Output(z,this,file_offset)
```

Comment:
As far as I can tell, this older routine `diagnostic1_Output` is no longer used, so modifying it here shouldn't cause any problems. It is just what is needed for per-bunch output.